sections/rich-text.liquid

```diff
@@ -12,7 +12,7 @@
           {%- when 'heading' -%}
             <h2 class="{{ block.settings.heading_size }} {{ block.settings.heading_align }}" {{ block.shopify_attributes }}>{{ block.settings.heading | escape }}</h2>
           {%- when 'text' -%}
-            <div class="rich-text__text rte {{ block.settings.text_align }}" {{ block.shopify_attributes }}>{{ block.settings.text }}</div>
+            <div class="rich-text__text rte {{ block.settings.text_align }}{% if block.settings.hyphenation %} hyphen{% endif %}" {{ block.shopify_attributes }} lang="de">{{ block.settings.text }}</div>
```

Comment:
Auch wenn wir absehbar nur deutsch schreiben, wäre es nicht vielleicht gut, die Sprache als Setting (mit DE als default) zu machen? 
Sonst schreibt man vielleicht in einem Jahr doch mal einen Text auf Englisch und wundert sich dann warum die Silbentrennung nicht richtig funnktioniert.
Man könnte auch das Hyphenation-Setting als Dropdown machen und als Optionen "Off", "English", "German".

Reply:
Gute Idee. Wird gemacht.